system-test/spanner.ts

```diff
@@ -4876,6 +4877,25 @@ describe('Spanner', () => {
         queryStreamMode(done, PG_DATABASE, query, POSTGRESQL_EXPECTED_ROW);
       });
 
+      // test for BATCH WRITE
```

Comment:
Is this feature not available for PGSQL ? If available add a test for PG/

Reply:
Done